internal/gatewayapi/translator.go

```diff
@@ -451,6 +451,9 @@ func (t *Translator) ProcessHTTPRoutes(httpRoutes []*v1beta1.HTTPRoute, gateways
 	var relevantHTTPRoutes []*HTTPRouteContext
 
 	for _, h := range httpRoutes {
+		if h == nil {
```

Comment:
same as https://github.com/envoyproxy/gateway/pull/216/files#r952032139

Reply:
Same for me, I'd rather see a `panic` than a `continue`.